VirtoCommerce.MarketingModule.Web/ExportImport/MarketingExportImport.cs

```diff
@@ -36,21 +36,21 @@ public BackupObject()
 
     public class MarketingExportImport
     {
-        private readonly IPromotionSearchService _promotionSearchService;
-        private readonly IDynamicContentSearchService _dynamicContentSearchService;
-        private readonly IPromotionService _promotionService;
-        private readonly ICouponService _couponService;
-        private readonly IDynamicContentService _dynamicContentService;
-        private readonly IPromotionUsageService _usageService;
+        protected readonly IPromotionSearchService promotionSearchService;
```

Comment:
Revert this commit - inherited classes can have their own private fields.
Also protected fields are not allowed.
Also protected member names should start with capital letter.

Reply:
Done